pkg/service/replay/replay.go

```diff
@@ -540,7 +554,12 @@ func (r *Replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 	var exitLoop bool
 	// var to store the error in the loop
 	var loopErr error
-
+	testSet, err := r.TestSetConf.Read(ctx, testSetID)
```

Comment:
Why are you not taking `conf` the passed argument but reading again from the db?